lib/cisco_node_utils/cmd_ref/DEPRECATED.yaml

```diff
@@ -0,0 +1,95 @@
+# DEPRECATED
```

Comment:
Feels a little too general.  Should we use something a bit more descriptive here like you did for `interface.DEPRECATED.yaml` ?